src/main/java/seedu/address/model/person/Avatar.java

```diff
@@ -0,0 +1,45 @@
+package seedu.address.model.person;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+
+import javafx.scene.image.Image;
+
+/**
+ * Represents displayed photo of a contact in the address book.
+ */
+public class Avatar {
+
+    private static final String DEFAULT_PATH = "/images/default_photo.png";
+    private String path;
+
+    public Avatar() {
+        this.path = DEFAULT_PATH;
+    }
+
+    public Avatar(String path) {
```

Comment:
As per your implementation, seems like this constructor is used when creating the newly edited person after `update photo` command.

A suggestion is to change this method will be

```
public Avatar(String path) throws FileNotFoundException {
    Image image = new Image(new FileInputStream(path));
}
```

Subsequently, your private method of `copyPerson` in your `UpdatePhotoCommand` class will throw `FileNotFoundException` and will be caught by its `execute` method. In the catch block, can consider the following

```
catch (FileNotFoundException e) {
    throw new CommandException("Invalid file path provided.");
}
```